docs/source/reference/indexing.rst

```diff
@@ -102,6 +102,7 @@ Conversion
 
    Index.astype
    Index.item
+   Index.to_list
```

Comment:
Do we exclude "tolist" on purpose?

Reply:
Yes, because pandas also exclude `tolist` in their docs.

Reply:
Sounds good, thanks!